src/app/core/services/electron/electron.service.ts

```diff
@@ -66,8 +66,36 @@ export class ElectronService {
     return result;
   }
 
-  async savePartsToStatistic(parts: PartWorkbook, dateReport: Date) {
-    const result = await this.ipcRenderer.invoke('saveParts', parts, dateReport);
-    return result;
+  async savePartsToStatistic(parts: PartWorkbook, dateReport: Date): Promise<void> {
+    const partsToSave = [];
+
+    for (const part of parts) {
```

Comment:
I'm pretty sure you didn't think that writing as much is efficient right ? 
Keep it DRY (Don't repeat yourself)

this can be replaced with something like this:
```
    const partsToSave = parts.map(part =>
      Object.entries(part).reduce((acc, [key, cell]) => {
        acc[key] = cell.value;
        return acc;
      }, {})
    );
```


Though it may quite a lot to unpack you can always move the reduce part to separate function